core/domain/story_services_test.py

```diff
@@ -515,11 +559,11 @@ def test_get_completed_nodes_in_story(self):
     def test_get_pending_nodes_in_story(self):
         self._record_completion(self.owner_id, self.STORY_1_ID, self.NODE_ID_1)
 
-        for _, pending_node in enumerate(
+        for index, pending_node in enumerate(
                 story_services.get_pending_nodes_in_story(
-                    self.owner_id, self.STORY_1_ID)):
+                    self.owner_id, self.STORY_1_ID), 1):
```

Comment:
Write `start=1` instead of `1` because that arg has a default value.

That said, this seems odd. Just use index + 1 below perhaps (and explain what's going on in a comment if you think it may be confusing).

Reply:
Done. I believe this should be straightforward because it's using enumerate, which is also used by many other tests in the codebase.

Reply:
Sorry, what do you mean by "this"?

Reply:
I meant using enumerate here makes sense right?

Reply:
Oh, by "it may be confusing" I meant the use of the "index + 1" instead of "index".